src/Facebook/InstantArticles/Transformer/Warnings/validator_warning_messages.ini

```diff
@@ -0,0 +1,42 @@
+;
+; This file contains messages for the transformer and also validator structure of an InstantArticle
+;
+
+[warning_messages]
+
+Ad = "Ad must have an URL or html body."
+Analytics = "Analytics must have an URL or html body."
+Anchor = "Anchor must have an URL on href property."
+AnimatedGIF = "GIF must have gif URL link."
+Audio = "Audio must have audio URL link."
+Author = "Author must have name."
+Blockquote = "Blockquote must contain at least one TextContainer or a plain text content."
+Bold = "This is a text container that must not be empty and not only contain only spaces, tab char, new line nor &nbsp;."
+Caption = "Caption must contain at least one TextContainer or a plain text content."
+Cite = "This is a text container that must not be empty and not only contain only spaces, tab char, new line nor &nbsp;."
+Div = "This is a text container that must not be empty and not only contain only spaces, tab char, new line nor &nbsp;."
+Footer = "Footer must have at least one of those: credits and/or copyright and/or relatedArticles."
+FormattedText = "This is a text container that must not be empty and not only contain only spaces, tab char, new line nor &nbsp;."
+GeoTag = "GeoTag must have script."
+H1 = "This is a text container that must not be empty and not only contain only spaces, tab char, new line nor &nbsp;."
+H2 = "This is a text container that must not be empty and not only contain only spaces, tab char, new line nor &nbsp;."
+H3 = "This is a text container that must not be empty and not only contain only spaces, tab char, new line nor &nbsp;."
+Header = "Header must have at least title and/or Ad."
+Image = "Image must have URL."
+InstantArticle = "InstantArticle relies on InstantArticleValidator." isValid method is not important."
+Interactive = "Interactive must have src and/or html."
+Italic = "This is a text container that must not be empty and not only contain only spaces, tab char, new line nor &nbsp;."
+LineBreak = "LineBreak is always valid."
+ListElement = "ListElement must have all of its ListItem valid."
```

Comment:
Adapt the language according to the changes if you make them. In this case: `ListElement must have at least one valid ListItem.`